core/block_render_svg_vertical.js

```diff
@@ -551,6 +606,19 @@ Blockly.BlockSvg.prototype.renderDraw_ = function(iconWidth, inputRows) {
     inputRows.rightEdge = Math.max(inputRows.rightEdge, 100);
   }
 
+  // Amount of space to skip drawing the top and bottom,
+  // to make room for the left and right to draw shapes (curves or angles).
+  this.edgeShapeWidth_ = 0;
+  this.edgeShape_ = null;
+  if (this.outputConnection) {
+    // Width of the curve/pointy-curve
+    var shape = this.outputConnection.getOutputShape();
```

Comment:
Do you mean the shape specifically? That shouldn't change, although edgeShapeWidth_ could change. I can move the calculation of edgeShape_ somewhere better.